integration/spark2/src/main/commonTo2.2And2.3/org/apache/spark/sql/hive/CarbonInMemorySessionState.scala

```diff
@@ -18,6 +18,7 @@ package org.apache.spark.sql.hive
 
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.fs.Path
+import org.apache.hadoop.hive.ql.metadata.HiveException
```

Comment:
scalastyle is passing for me in local too, initially also there was no space

Reply:
I think there should be space between different system imports, like there shall be space between carbon and spark system, same shall be followed in case of hadoop also, spark is following the same styling format. please have a look, i think you can revisit the scala style guideline once again and update if required.
![image](https://user-images.githubusercontent.com/12999161/46254013-62712e80-c4a6-11e8-8c77-006b5f2565c5.png)